kore/src/Kore/Simplify/API.hs

```diff
@@ -12,24 +12,6 @@ Portability : portable
 module Kore.Simplify.API (
     evalSimplifier,
     evalSimplifierProofs,
-
-    -- * Re-exports
-    Env (..),
-    Simplifier,
-    runSimplifier,
```

Comment:
Maybe we should rename the module (again :thinking: ) then?
An `API` module will probably always have a small number of re-exports...

Reply:
yeah, if you look at the changes, it was always just like 3 `Kore.Simplify.Simplify` types/functions that were being exported...most of the re-exports were redundant anyway... given the functions in this module maybe `Kore.Simplify.Env` would be a better name now?